st2common/st2common/config.py

```diff
@@ -94,6 +94,12 @@ def register_opts(ignore_errors=False):
         cfg.StrOpt('db_name', default='st2', help='name of database'),
         cfg.StrOpt('username', help='username for db login'),
         cfg.StrOpt('password', help='password for db login'),
+        cfg.IntOpt('connection_retry_max_delay_m', help='Connection retry total time (minutes).',
```

Comment:
Yeah, I wish we would use the same consistent setting names and API everywhere where we implement retries (message bus, querier, mistral, ...).

Otherwise it makes for a inconsistent and confusing interface...